Rebuilt quake for sound support

ChinYikMing opened the pull request:
The prebuilt Quake binary has been updated to reflect the recent addition of sound related system calls.

jserv reviewed (changes requested):
Make the git commit message informative.
See https://github.com/sysprog21/rv32emu/commit/7ab4b6c51ed803bb479d80979a7a024463d1dba2

ChinYikMing committed:
Rebuilt quake for sound support

The prebuilt Quake binary has been updated to reflect the recent
addition of sound related system calls.

commit: fc733cd
